examples/3d-tiles/generate-index/args.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
```

Comment:
Maybe a `scripts` folder.

Wonder if this can be generalized and placed in root `scripts`.

Reply:
Right now the scripts need be transpiled. I don't want to introduce all the dependencies to root, so that's why I keep the scripts in example.